src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1441,6 +1458,22 @@ ProviderTypes.Spotify or ProviderTypes.StackExchange or ProviderTypes.Zoom
                 // Typeform returns the username as a custom "alias" node:
                 ProviderTypes.Typeform => (string?) context.UserInfoResponse?["alias"],
 
+                // VK ID returns the username as a custom "user/first_name" and "user/last_name" nodes
+                // (https://id.vk.com/about/business/go/docs/en/vkid/latest/vk-id/connection/api-integration/api-description#App-access-to-user-data):
+                ProviderTypes.VkId 
```

Comment:
Same here: you can merge that line with the existing `ProviderTypes.Basecamp or ProviderTypes.Harvest` cases.